WordPress/src/main/java/org/wordpress/android/ui/pages/PageItemViewHolder.kt

```diff
@@ -80,8 +81,19 @@ sealed class PageItemViewHolder(internal val parent: ViewGroup, @LayoutRes layou
                     page.title
 
                 val date = if (page.date == Date(0)) Date() else page.date
-                time.text = DateTimeUtils.javaDateToTimeSpan(date, parent.context)
+                val stringDate = DateTimeUtils.javaDateToTimeSpan(date, parent.context)
                         .capitalizeWithLocaleWithoutLint(parent.context.currentLocale)
+                val subtitle = page.subtitle
+                pageSubtitle.text = if (subtitle == null) {
+                    stringDate
+                } else {
+                    String.format(
+                            Locale.getDefault(),
```

Comment:
I've seen we have a very inconsistent use of `getLocale()` throughout the app.
Sometimes we use `Locale.US`, sometimes we sue `String.format()` without indicating the Locale, sometimes `Locale.getDefault()` as you use here, and sometimes we even have our own `LanguageUtils` function that does exactly that, called `getCurrentDeviceLanguage`.

This is just a comment as I was curious to see if there was a particular "way we do things"  in the codebase but it seems to be all evenly split. No action need be taken but thought I'd still make the observation here, in case this triggers any thoughts.

Reply:
I'm honestly not sure why this method needs a Locale, the documentation says it's for localization. I think we should always use `getDefault` for user facing texts. We should use Locale.US if we want a consistent result (that's not translated) - this is for example for tracking or for API calls